tests/installation/partitioning_smalldisk_storageng.pm

```diff
@@ -0,0 +1,88 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test to take the smallest disk in storage_ng scenarios
+#    Otherwise the partitioning proposal will use a free disk, which makes
+#    rebooting a game of chance on real hardware
+# Maintainer: Alvaro Carvajal <acarvajal@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+use version_utils qw(is_storage_ng);
+use partition_setup qw(take_first_disk);
+
+sub run {
+    if (!is_storage_ng) {
+        record_info "Requires storage_ng", "This module only works with storage_ng which is not present. Selecting first disk instead";
+        return take_first_disk;
+    }
+
+    select_console 'root-ssh';
+    my $lsblkcmd = q/echo "[$(lsblk -n -l -o SIZE,NAME -d -e 7,11 -b | sort -n | head -1 | awk '{print $2}')]"/;
```

Comment:
`| head -1 | awk '{print $2}'` could be replaced by `| awk 'NR==1 {print $2}'`. It's always better in sh scripts to reduce the number of pipe! Sometimes order of pipes are changed! (really, we had this in my previous company),

Reply:
Sure. Will do that.

Reply:
Verification run: http://mango.qa.suse.de/tests/3200#step/partitioning_smalldisk_storageng/5